rmgpy/data/kinetics/family.py

```diff
@@ -1519,10 +1519,9 @@ def apply_recipe(self, reactant_structures, forward=True, unique=True, relabel_a
             # If product structures are Group objects and the reaction is in certain families
             # (families with charged substances), the charge of structures will be updated
             if isinstance(struct, Molecule):
-                struct.update_charge()
```

Comment:
Is it possible for us to instead skip updating charge for just the cutting labels and not the whole fragment?